ads/_config.js

```diff
@@ -15,235 +15,313 @@
+      'https://static.industrybrains.com',
+    ],
+  },
+
+  inmobi: {
+    prefetch: 'https://cf.cdn.inmobi.com/ad/inmobi.secure.js',
+    renderStartImplemented: true,
+  },
+
+  mads: {
+    prefetch: 'https://eu2.madsone.com/js/tags.js',
+  },
+
+  'mantis-display': {
+    prefetch: 'https://assets.mantisadnetwork.com/mantodea.min.js',
+    preconnect: [
+      'https://mantodea.mantisadnetwork.com',
+      'https://res.cloudinary.com',
+      'https://resize.mantisadnetwork.com',
+    ],
+  },
+
+  'mantis-recommend': {
+    prefetch: 'https://assets.mantisadnetwork.com/recommend.min.js',
+    preconnect: [
+      'https://mantodea.mantisadnetwork.com',
+      'https://resize.mantisadnetwork.com',
+    ],
+  },
+
+  mediaimpact: {
+    prefetch: 'https://ec-ns.sascdn.com/diff/251/divscripte/amp.js',
+  },
+
+  microad: {
+    prefetch: 'https://j.microad.net/js/camp.js',
+    preconnect: [
+      'https://s-rtb.send.microad.jp',
+      'https://s-rtb.send.microadinc.com',
+      'https://cache.send.microad.jp',
+      'https://cache.send.microadinc.com',
+      'https://deb.send.microad.jp',
+    ],
+  },
+
+  nativo: {
+    prefetch: 'https://s.ntv.io/serve/load.js',
+  },
+
+  nend: {
+    prefetch: 'https://js1.nend.net/js/amp.js',
+    preconnect: [
+      'https://output.nend.net',
+      'https://img1.nend.net',
+    ],
+  },
+
+  openx: {
+    prefetch: 'https://www.googletagservices.com/tag/js/gpt.js',
+    preconnect: [
+      'https://partner.googleadservices.com',
+      'https://securepubads.g.doubleclick.net',
+      'https://tpc.googlesyndication.com',
+    ],
+  },
+
+  pubmatic: {
+    prefetch: 'https://ads.pubmatic.com/AdServer/js/amp.js',
+  },
+
+  pubmine: {
+    prefetch: 'https://s.pubmine.com/showad.js',
+  },
+
+  pulsepoint: {
+    prefetch: 'https://ads.contextweb.com/TagPublish/getjs.static.js',
+    preconnect: 'https://tag.contextweb.com',
+  },
+
+  revcontent: {
+    prefetch: 'https://labs-cdn.revcontent.com/build/amphtml/revcontent.amp.min.js',
+    preconnect: [
+      'https://trends.revcontent.com',
+      'https://cdn.revcontent.com',
+      'https://img.revcontent.com',
+    ],
+  },
+
+  smartadserver: {
+    prefetch: 'https://ec-ns.sascdn.com/diff/js/amp.v0.js',
+    preconnect: 'https://static.sascdn.com',
+    renderStartImplemented: true,
+  },
+
+  sortable: {
+    prefetch: 'https://www.googletagservices.com/tag/js/gpt.js',
+    preconnect: [
+      'https://tags-cdn.deployads.com',
+      'https://partner.googleadservices.com',
+      'https://securepubads.g.doubleclick.net',
+      'https://tpc.googlesyndication.com',
+    ],
+  },
+
+  sovrn: {
+    prefetch: 'https://ap.lijit.com/www/sovrn_amp/sovrn_ads.js',
+  },
+
+  teads: {
+    prefetch: 'https://cdn.teads.tv/media/format/v3/teads-format.min.js',
+    preconnect: [
+      'https://cdn2.teads.tv',
+      'https://a.teads.tv',
+      'https://t.teads.tv',
+    ],
+  },
+
+  'weborama-display': {
```

Comment:
we know type name must be a string. that's why we don't need to write as `weborama-display` explicitly. All other type names are like 
```
doubleclick: {
}
```
instead of 
```
'doubleclick': {
}
```
This is one special case, maybe we should fix it